spec/liquidityCalculator.ts

```diff
@@ -49,6 +49,7 @@ describe('Calculate liquidity', () => {
         creationDate: moment('2016-11-10'),
         isSold: true,
         kms: adFields.KMS,
+        lastTimeUpDate: moment('2016-11-10'),
```

Comment:
moment call returns a moment object not a date. is it OK for us in this case? shouldn't we call .toDate() here?

Reply:
It works correctly